.gitignore

```diff
@@ -88,4 +88,7 @@ typings/
 .fusebox/
 
 # DynamoDB Local files
-.dynamodb/
\ No newline at end of file
+.dynamodb/
+/.idea
+/root
+/html
```

Comment:
You'll want to revert thoses changes

Reply:
Yes! Sorry this was an overlook on my part 😅

Thank your for mentioning it though!